client/src/pages/SignUp/SignUp.tsx

```diff
@@ -37,24 +38,43 @@ export default function Register(): JSX.Element {
     });
   };
 
+  const handleDemoLogin = () => {
+    login('jondoe@vmail.com', '12345678').then((data) => {
+      if (data.error) {
+        // setSubmitting(false);
```

Comment:
Same here, also, if this code is identical then we are not following DRY principles. We should be keeping our codebase more maintainable and have this available for both Sign up and Login but only one component is necessary for this.